host/src/attribute_server.rs

```diff
@@ -371,19 +369,23 @@ impl<'d, M: RawMutex, const MAX: usize> AttributeServer<'d, M, MAX> {
         Ok(w.len())
     }
 
-    fn handle_read_blob(&self, buf: &mut [u8], handle: u16, offset: u16) -> Result<usize, codec::Error> {
+    fn handle_read_blob(
+        &self,
+        connection: &Connection,
```

Comment:
some places this is conn, others connection

Reply:
Yeah the places with `connection` are the ones I added. I'll align them all for neatness.

Reply:
95775811e30216eb1518a0688fe5f81e527f22f2